modules/storage-api/src/testFixtures/java/org/apache/ignite/internal/storage/AbstractMvTableStorageTest.java

```diff
@@ -123,25 +126,24 @@ public abstract class AbstractMvTableStorageTest extends BaseMvStoragesTest {
 
     protected StorageHashIndexDescriptor hashIdx;
 
+    protected StorageIndexDescriptor pkIdx;
+
     private final CatalogService catalogService = mock(CatalogService.class);
 
-    protected final StorageIndexDescriptorSupplier indexDescriptorSupplier = new StorageIndexDescriptorSupplier() {
-        @Override
-        public @Nullable StorageIndexDescriptor get(int indexId) {
-            int catalogVersion = catalogService.latestCatalogVersion();
+    protected final StorageIndexDescriptorSupplier indexDescriptorSupplier = indexId -> {
```

Comment:
Another useless code change. What was your motivation behind it?

Reply:
Lambda takes fewer lines of code, looks more modern and elegant, are you against it?

Reply:
Generally speaking, I'm against changing the code that's already good enough. And we we really want to do so, it's better to be done centralized.
That anonymous class wasn't bad. Few lines of extra code are not a real issue. So I conclude that you just wanted to change something because it didn't look like you wanted it to look, instead of thinking "does it really look inappropriate to the point of re-writing it?". I don't like useless changes in code. They bloat your pull requests, artificially increasing their sizes. They don't really contribute to the quality of the product - code doesn't become better.
_We should concentrate our efforts on real issues, instead of changing the code that's already good._

Reply:
Answered a little higher.

Reply:
Reverts changes.